### PR TITLE
Allow usage in Laravel5.5

### PR DESCRIPTION
**Pull request by m1guelpf**
Laravel 5.5 isn't released yet, but lots of people (including me) are already using it. This pull request enables the usage of this package in Laravel 5.5

Also, don't forget to tag a new release after merging!

:octocat:

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,9 +26,9 @@
     ],
     "require": {
         "php" : "^7.0",
-        "illuminate/config": "~5.1.0|~5.2.0|~5.3.0|~5.4.0",
-        "illuminate/database": "~5.1.0|~5.2.0|~5.3.0|~5.4.0",
-        "illuminate/support": "~5.1.0|~5.2.0|~5.3.0|~5.4.0",
+        "illuminate/config": "~5.1.0|~5.2.0|~5.3.0|~5.4.0|~5.5.0",
+        "illuminate/database": "~5.1.0|~5.2.0|~5.3.0|~5.4.0|~5.5.0",
+        "illuminate/support": "~5.1.0|~5.2.0|~5.3.0|~5.4.0|~5.4.0",
```
Comment (freekmurze) [inline composer.json]: Typo here, should be `5.5.0`

**Comment by freekmurze** [comment]
I'm going to wait to pull this in until L5.5 is officially available.

**Comment by m1guelpf** [comment]
@freekmurze Fixed!

**Comment by m1guelpf** [comment]
@freekmurze And what about early users? I can't see anything you can lose upgrading...

**Comment by freekmurze** [comment]
Early users can fork the package and make the necessary adjustments in that fork. I simply don't have the time to update or keep track of which packages need updating before L5.5 is released.

I prefer to tag L5.5 compatible versions of all our packages in one go shortly after L5.5 is publicly available.

**Pull request closed without merging**